client/ayon_houdini/api/lib.py

```diff
@@ -1283,7 +1283,7 @@ def is_attached_to_node(image):
         return attached_images
 
 
-def find_active_network(category, default):
+def find_active_network(category=None, default="/obj"):
```

Comment:
Why was this change needed for this PR. I'd rather keep these explicitly required instead of 'falliing back' to some arbitrary default.